middleware/middleware.js

```diff
@@ -70,9 +115,32 @@ module.exports.fetchRandomGameDataArr = async (req, res) => {
   const response = await this.fetchFromApi(maxPage, pageSize);
 
   if (response.status === 200) {
+    //add a conditional to see if the game data is already in mongodb
+    const alreadyExists = await GameData.findOne({
```

Comment:
Not sure about this name, data returned is an object?

Reply:
I guess the naming could be better, but it should either be empty or return a game object.